src/vendor/flickity.js

```diff
@@ -0,0 +1,8 @@
+var flkty = new Flickity('.main-carousel', {
```

Comment:
Ten kod jest tylko kodem inicjującym karuzelę i powinien być po prostu w folderze `scripts` tak jak inne pliki, - coś się nie zrozumieliśmy wcześniej.

- dodaj 'use strict` 
- bez `var`
- usuń komentarz

Reply:
Zrobiłem tak ale dalej wychodził error z eslintem przy commicie więc wrzuciłem flickity.js z powrotem do vendor.  Proszę o merge

Reply:
To nie jest rozwiązanie,

Jednak skoro nic więcej z tą karuzelą nie robisz poza jej inicjalizacją to zamiast pisać kawałek kodu js, możesz wykorzystać do tego celu inicjalizacje przez html 
https://flickity.metafizzy.co/#initialize-with-html

A żeby zostawić ten kod, to potrzebujesz wskazać dla eslinta, że `Flickity` jest globalnym obiektem https://eslint.org/docs/user-guide/configuring#specifying-globals, a ponieważ nie korzystasz ze zmiennej `flkty` to dostajesz błąd. ale użycie tej zmiennej tutaj jest nie potrzebne.